hugegraph-server/hugegraph-test/src/main/resources/hugegraph.properties

```diff
@@ -17,8 +17,9 @@
 
 gremlin.graph=org.apache.hugegraph.HugeFactory
 
-backend=${backend}
-serializer=${serializer}
+backend=hstore
+serializer=binary
+task.scheduler_type=local
```

Comment:
The ${} here is used to replace variables for testing different backends, and no modification is necessary :)